src/Frontend/src/views/throughputreport/setup/ConnectionSetupView.vue

```diff
@@ -0,0 +1,118 @@
+<script setup lang="ts">
+import { computed, onMounted, ref } from "vue";
+import ThroughputConnectionSettings from "@/resources/ThroughputConnectionSettings";
+import { Transport } from "@/views/throughputreport/transport";
+import throughputClient from "@/views/throughputreport/throughputClient";
+import { useIsMonitoringEnabled } from "@/composables/serviceServiceControlUrls";
+import ConfigurationCode from "@/views/throughputreport/setup/ConfigurationCode.vue";
+import { useThroughputStore } from "@/stores/ThroughputStore";
+import { storeToRefs } from "pinia";
+
+const store = useThroughputStore();
+const { testResults } = storeToRefs(store);
+const settingsInfo = ref<ThroughputConnectionSettings | null>(null);
+const transport = computed(() => {
+  if (testResults.value == null) {
+    return Transport.None;
+  }
+
+  return testResults.value.transport as Transport;
+});
+
+onMounted(async () => {
+  settingsInfo.value = await throughputClient.setting();
+});
+
+function transportNameForInstructions() {
```

Comment:
is there a reason we can't use the value from ThroughputStore here which does the same thing?

Reply:
good catch, fixed